include/arcade/Color.hpp

```diff
@@ -0,0 +1,100 @@
+/*
+** EPITECH PROJECT, 2022
+** Arcade
+** File description:
+** Color
+*/
+
+/// @file
+///
+/// The Color data structure.
+
+#ifndef ARCADE_COLOR_HPP_
+#define ARCADE_COLOR_HPP_
+
+#include <cstddef>
+#include <cstdint>
+
+namespace arcade
+{
+    /// 8bits colors used as backup in IGameObject interface (setForeground, setBackground).
+    enum class DefaultColor { Black, White, Transparent, Red, Green, Blue, Yellow, Magenta, Cyan };
```

Comment:
It ultimately a matter of personal preference, but I prefer seperate symbols to be in their own separate lines.